### PR TITLE
Skills Clarity Update

### PR DESCRIPTION
**Pull request by rosebrallier**
Edits re: [Issue #517 ](https://github.com/bleehu/Compound_X/issues/517)

### CODE REVIEW

File: 01_01_Basic_Rules.txt [hunk 1]
```diff
@@ -425,6 +421,19 @@ class has a Proficiency with a given armor type then they can purchase or wear
 that armor type in combat. If your character doesn't have that proficiency, then
 they can't.
 
+===== Assigning Skills to a Character =====
+
+	Characters gain points in a wide variety of skills to represent their experience
+and expertise in different fields and activities. Every time your character levels up,
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt:427]: Oooo prose 👍

File: 01_01_Basic_Rules.txt [hunk 2]
```diff
@@ -425,6 +421,19 @@ class has a Proficiency with a given armor type then they can purchase or wear
...
+===== Assigning Skills to a Character =====
+
+	Characters gain points in a wide variety of skills to represent their experience
+and expertise in different fields and activities. Every time your character levels up,
+they gain skill points equal to double your Skill Point Gain. Assign each skill point to
+a skill as you see fit; see CharacterCreation/06_Skills for details about skills and for
+a list of commonly-used skills in Compound X. At level 1, your character will have two
+times their Skill Point Gain to assign at will. You may gain a maximum of 60 skill
+points in a single skill. In addition, each time you level up, you may gain a maximum of
+6 points in any one skill by using your Skill Point Gain. Skill gained from Feats, Class
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt]: 'using your Skill Point Gain' sounds strange to me, and I'm not certain that it's necessary to the sentence. Nonblocking

File: 01_01_Basic_Rules.txt [hunk 3]
```diff
@@ -425,6 +421,19 @@ class has a Proficiency with a given armor type then they can purchase or wear
...
 they can't.
 
+===== Assigning Skills to a Character =====
+
+	Characters gain points in a wide variety of skills to represent their experience
+and expertise in different fields and activities. Every time your character levels up,
+they gain skill points equal to double your Skill Point Gain. Assign each skill point to
+a skill as you see fit; see CharacterCreation/06_Skills for details about skills and for
+a list of commonly-used skills in Compound X. At level 1, your character will have two
+times their Skill Point Gain to assign at will. You may gain a maximum of 60 skill
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt]: I am concerned that we may be sending mixed signals by putting "Assign ... as you see fit" and "to assign at will" before the restrictions of the 60 point cap and 6 points per level cap. Nonblocking.

**Review by Turtlelord26 — APPROVED** [review]
🐢 likes.

File: 01_01_Basic_Rules.txt (hunk 2 above)
Comment (rosebrallier) [reply 01_01_Basic_Rules.txt]: Good point

File: 01_01_Basic_Rules.txt (hunk 3 above)
Comment (rosebrallier) [reply 01_01_Basic_Rules.txt]: Yeah, I was iffy on that while writing it

**Review by bleehu — APPROVED** [review]
shipit!

**Pull request merged**